install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -650,6 +650,19 @@ enableK8sEventHandover: false
 # -- Enable CiliumEndpointSlice feature.
 enableCiliumEndpointSlice: false
 
+envoyConfig:
+  # -- Enable envoy config CRD
+  # Cilium Envoy Config CRD can also be implicitly enabled by other options.
+  enabled: false
```

Comment:
suggestion: set this value to `~` so we are free to update the defaulting later.

Reply:
Done.

Reply:
From an end-user, I would expect to be able to set `foo.enabled=false` and have that disable foo. But `envoyConfig.enabled=false` doesn't disable `envoyConfig`. Rather, it sets it to "default".

I would expect a value of `~` to mean "please compute a reasonable default".

That said, if this is consistent with how we've exposed other variables, I'm not going to be picky.